src/cli.ts

```diff
@@ -115,6 +116,12 @@ function makeCommand (process: NodeJS.Process): Command {
     ).env(
       'BOM_REPRODUCIBLE'
     )
+  ).addOption(
+    new Option(
```

Comment:
please also add this CLI option to the README

Reply:
Is there: https://github.com/CycloneDX/cyclonedx-node-npm/pull/1243/files#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5